ops/model.py

```diff
@@ -789,26 +789,28 @@ class _ModelBackendValidator:
     @classmethod
     def validate_metric_key(cls, key):
         if cls.METRIC_KEY_REGEX.match(key) is None:
-            raise ModelError(f'invalid metric key {repr(key)}: must start with [a-zA-Z] and contain [a-zA-Z0-9_] only')
+            raise ModelError(f'invalid metric key {repr(key)}: must match {cls.METRIC_KEY_REGEX.pattern}')
```

Comment:
the only problem with this is that it isn't clear if it communicates well to a human trying to do their development, the counterpoint is that it is guaranteed to be correct. :)

Reply:
Agreed, I tried to find a short way to formulate this but having a regex seems more precise and less verbose.